package.json

```diff
@@ -7,7 +7,7 @@
     "test": "test"
   },
   "scripts": {
-    "clean": "rm -rf dist",
+    "clean": "rm -rf dist && rm -rf reports",
```

Comment:
you can pass a list of folder names to delete, separate by space
you need only one `rm -rf`